SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -134,29 +134,29 @@ def updateMRMLFromGUI(self):
     self.scriptedEffect.setParameter("NeighborhoodSizeMm", self.neighborhoodSizeMmSlider.value)
     self.scriptedEffect.parameterSetNode().SetNodeReferenceID("FloodFilling.ROI", self.roiSelector.currentNodeID)
 
-  def getClippedMasterImageData(self):
-    # Return masterImageData unchanged if there is no ROI
-    masterImageData = self.scriptedEffect.masterVolumeImageData()
+  def getclippedSourceImageData(self):
+    # Return sourceImageData unchanged if there is no ROI
+    sourceImageData = self.scriptedEffect.sourceVolumeImageData()
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.